spec/rubocop/config_loader_spec.rb

```diff
@@ -292,6 +292,8 @@
       expect(configuration).to eq({})
     end
 
+    # TODO: figure out why these specs print a green dot even when they fail
+    # (probably related to the process shenanigans)
```

Comment:
this is a bit concerning, I think, but I have NO idea how to resolve it :grimacing: